package.json

```diff
@@ -13,11 +15,15 @@
   "author": "CAR FOR YOU",
   "homepage": "https://github.com/autoricardo/carforyou-phraseapp-pkg#readme",
   "devDependencies": {
+    "@carforyou/tslint-rules": "0.0.5",
     "@pika/pack": "^0.3.7",
     "@pika/plugin-build-node": "^0.3.16",
     "@pika/plugin-copy-assets": "^0.3.16",
     "@pika/plugin-simple-bin": "^0.3.16",
-    "@pika/plugin-standard-pkg": "^0.3.16"
+    "@pika/plugin-standard-pkg": "^0.3.16",
+    "prettier": "^1.17.0",
+    "tslint": "^5.16.0",
+    "typescript": "^3.4.5"
```

Comment:
Sidenote, do we write those in typescript? Shouldn't we?
It seems a bit of an overkill to add typescript just for the lint rules...

Reply:
it's a hard peer dependency of tslint. we can't go without it. but it's also just a devDependency. i _think_ it's fine